src/views/virtualmachines/actions/VirtualMachineActionFactory.tsx

```diff
@@ -126,7 +127,10 @@ export const VirtualMachineActionFactory = {
           <DeleteVMModal isOpen={isOpen} onClose={onClose} vm={vm} />
         )),
       description: isRunning(vm) && t('The VirtualMachine is running'),
-      disabled: isRunning(vm),
+      disabled: isRunning(vm) || isDeletionProtectionEnabled(vm),
```

Comment:
do we display the tooltip if VM is running?

Reply:
No. It should only show if the deletion protection is turned on and the VM is off. This is clearer with the updated implementation.

Reply:
not sure if we can use falsy values for a tooltip. Just in case I would pass `undefined`  explicitly i.e. 
 ```
isDeletionProtectionEnabled() ? t('foo') : undefined
```

Reply:
There are checks that prevent it from being used if it's falsy, but making it explicit is better. Fixed!